report/src/main/webapp/app/layout/toolbar/toolbarBottom.view.html

```diff
@@ -20,12 +20,14 @@
 <div class="toolbar-bottom">
   <div class="toolbar-link">
     <h4 class="toolbar-link-header ellipsis">
-        <i ng-if="toolbarBottom.viewMode != 'suite'" class="glyphicon glyphicon-screenshot"
+        <div ng-if="toolbarBottom.viewMode != 'suite'"class="fontawesome-link"
```

Comment:
Are you sure we need all those wrappers for popovers? I mean maybe just"i" tag instead of "div" with nested "i" was fine?